### ISSUE TITLE
NPM installation fails with phantomjs errors

### ISSUE THREAD

**Issue by iamaserverguy** [ston3o/netclix#20]
I am on openSUSE Tumbelweed - 20190320 and installation fails on Phantomjs.
Below are logs, how should I fix this error ?

```
576 info lifecycle phantomjs-prebuilt@2.1.16~install: phantomjs-prebuilt@2.1.16
3577 verbose lifecycle phantomjs-prebuilt@2.1.16~install: unsafe-perm in lifecycle false
3578 verbose lifecycle phantomjs-prebuilt@2.1.16~install: PATH: /usr/lib64/node_modules/npm10/node_modules/npm-lifecycle/node-gyp-bin:/usr/local/lib/node_modules/netclix/node_modules/phantomjs-prebuilt/node_modules/.bin:/usr/local/lib/node_modules/netclix/node_modules/.bin:/usr/local/lib/node_modules/.bin:/usr/sbin:/usr/bin:/sbin:/bin
3579 verbose lifecycle phantomjs-prebuilt@2.1.16~install: CWD: /usr/local/lib/node_modules/netclix/node_modules/phantomjs-prebuilt
3580 silly lifecycle phantomjs-prebuilt@2.1.16~install: Args: [ '-c', 'node install.js' ]
3581 silly lifecycle phantomjs-prebuilt@2.1.16~install: Returned: code: 1  signal: null
3582 info lifecycle phantomjs-prebuilt@2.1.16~install: Failed to exec install script
3583 timing action:install Completed in 3872ms
3584 verbose unlock done using /root/.npm/_locks/staging-3a08f0df5026584d.lock for /usr/local/lib/node_modules/.staging
3585 timing stage:rollbackFailedOptional Completed in 241ms
3586 timing stage:runTopLevelLifecycles Completed in 6578ms
3587 verbose stack Error: phantomjs-prebuilt@2.1.16 install: `node install.js`
3587 verbose stack Exit status 1
3587 verbose stack     at EventEmitter.<anonymous> (/usr/lib64/node_modules/npm10/node_modules/npm-lifecycle/index.js:301:16)
3587 verbose stack     at EventEmitter.emit (events.js:189:13)
3587 verbose stack     at ChildProcess.<anonymous> (/usr/lib64/node_modules/npm10/node_modules/npm-lifecycle/lib/spawn.js:55:14)
3587 verbose stack     at ChildProcess.emit (events.js:189:13)
3587 verbose stack     at maybeClose (internal/child_process.js:970:16)
3587 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
3588 verbose pkgid phantomjs-prebuilt@2.1.16
3589 verbose cwd /
3590 verbose Linux 5.0.2-1-default
3591 verbose argv "/usr/bin/node10" "/usr/bin/npm-default" "install" "--global" "netclix"
3592 verbose node v10.15.2
3593 verbose npm  v6.4.1
3594 error code ELIFECYCLE
3595 error errno 1
3596 error phantomjs-prebuilt@2.1.16 install: `node install.js`
3596 error Exit status 1
3597 error Failed at the phantomjs-prebuilt@2.1.16 install script.
3597 error This is probably not a problem with npm. There is likely additional logging output above.
3598 verbose exit [ 1, true ]


```

**Comment by johackim** [comment]
Hello @iamaserverguy, thanks for this issue.

Mabye try to use others node version than 10.15.2 with [nvm](https://github.com/creationix/nvm).
I personaly use nvm with 10.15.3 node version.

```bash
nvm install v10.15.3
nvm use v10.15.3
```

**Comment by iamaserverguy** [comment]
> Hello @iamaserverguy, thanks for this issue.
> 
> Mabye try to use others node version than 10.15.2 with [nvm](https://github.com/creationix/nvm).
> I personaly use nvm with 10.15.3 node version.
> 
> ```shell
> nvm install v10.15.3
> nvm use v10.15.3
> ```

Thank you, Issue has been fixed with nvm 10.15.3
